Dfe.Academies.Academisation.Domain/ApplicationAggregate/Trusts/JoinTrust.cs

```diff
@@ -24,19 +24,19 @@ private JoinTrust(int id, int UKPRN, string trustName, string trustReference, Ch
 			this.ChangesToLaGovernanceExplained = changesToLaGovernanceExplained;
 		}
 
-		public int UKPRN { get; private set; }
+		public int UKPRN { get;  set; }
```

Comment:
Please can you remind me why these need have changed?

Reply:
Ah It was when I was following the vertical slice convention as set out in the readme, now I'm following Tim's pattern in the ProjectAddNotes, I can move these back to private.